anaconda-project_template.yml

```diff
@@ -0,0 +1,39 @@
+# To reproduce: install 'anaconda-project', then 'anaconda-project run'
+name: NAME
+description: DESC
+maintainers:
+  - MAINTAINER
+
+channels:
+  - pyviz
+
+dependencies: &deps
+  - python=3.6
+  - notebook ==5.7.8
+  - ipykernel ==5.1.0
+  - nomkl
```

Comment:
Can you document why we suggest `nomkl`?

Reply:
We might want to talk about that actually.  I originally did it to speed up environment set up times. But we should probably decide as a group whether or not we want nomkl in all these projects @pyviz-topics/pyviz-dev

Reply:
Usually mkl is used for better runtime performance in numpy operations, but maybe with datashader that's not so important, as we use Numba for most of the internal computations anyway?